schema.graphql

```diff
@@ -74,6 +72,8 @@ type AllowedAssetToken @entity {
 
   assetToken: Bytes
   isAllowed: Boolean
+  assetDepositMin: BigInt
```

Comment:
This is not part of the AllowedAssetToken - completed unrelated.  Please move back